hack/README.md

```diff
@@ -10,7 +10,35 @@ This is the script to update clientset/informers/listers and API deepcopy code u
 
 Make sure to run this script after making changes to /pkg/apis/volumesnapshot/v1beta1/types.go.
 
-To run this script, simply run: ./hack/update-generated-code.sh from the project root directory.
+To run this script, you have to patch it:
```

Comment:
Can you add some words here to explain why we have to go thru these steps to generate code and reference this issue https://github.com/kubernetes-csi/external-snapshotter/issues/239.

Reply:
I'm going to merged this PR.
@alex1989hu can you submit a followup PR to address this comment.